core/src/test/java/org/ehcache/EhcacheBasicGetAllTest.java

```diff
@@ -2100,6 +3026,53 @@ public void testGetAllStoreAllMatchCacheAccessExceptionBeforeLoaderNoMatchSomeFa
     validateStats(ehcache, EnumSet.noneOf(CacheOperationOutcomes.GetOutcome.class));
     validateStats(ehcache, EnumSet.noneOf(CacheOperationOutcomes.CacheLoadingOutcome.class));
   }
+  
+  /**
+   * Tests {@link Ehcache#getAll(Set)} for
+   * <ul>
+   *    <li>non-empty request key set</li> 
+   *    <li>all {@link Store} entries match</li>
+   *    <li>{@link Store#bulkComputeIfAbsent} throws before accessing loader</li>
+   *    <li>no {@link CacheLoaderWriter} entries match</li>
+   *    <li>some {@link CacheLoaderWriter#loadAll(Iterable)} calls fail with {@link BulkCacheLoadingException}</li>
+   * </ul>
+   */
+  @Test
+  public void testGetAllStoreAllMatchCacheAccessExceptionBeforeLoaderNoMatchSomeFailWithBulkCacheLoadingException() throws Exception {
+    final FakeStore fakeStore = new FakeStore(getEntryMap(KEY_SET_A, KEY_SET_B));
+    this.store = spy(fakeStore);
+    doThrow(new CacheAccessException("")).when(this.store)
+        .bulkComputeIfAbsent(getAnyStringSet(), getAnyIterableFunction());
+
+    final FakeCacheLoaderWriter fakeLoader = new FakeCacheLoaderWriter(getEntryMap(KEY_SET_C, KEY_SET_F), KEY_SET_F, true);
```

Comment:
Do not start in an inconsistent state: the loader needs to know a super set of what the store knows. If that makes the test a duplicate, delete it.